config/routes.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+Rails.application.routes.draw do
+  resources :posts
```

Comment:
Please remove this line as it exposes all the routes for posts and you have already configured the necessary routes for posts below.